### PR TITLE
fix volume expand bug

### PR DESCRIPTION
**Pull request by peng9808**
#### What this PR does / why we need it:
fix volume expand bug
#### Special notes for your reviewer:
NONE
#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

File: pkg/apiserver/manager/hwameistor/localvolume_controller.go [hunk 1]
```diff
@@ -439,7 +440,7 @@ func (lvController *LocalVolumeController) CreateVolumeExpand(volName string, ta
 		return nil, err
 	}
 
-	if quantity.Value() > freeCount {
+	if quantity.Value() > freeCount+requiredCapacityBytes {
```
Comment (SSmallMonster) [inline pkg/apiserver/manager/hwameistor/localvolume_controller.go:443]: rename variable `quantity` to `targteCapacityBytes` may be easy to understand

**Review by SSmallMonster — CHANGES_REQUESTED** [review]

File: pkg/apiserver/manager/hwameistor/localvolume_controller.go (hunk 1 above)
Comment (peng9808) [reply pkg/apiserver/manager/hwameistor/localvolume_controller.go:443]: ok

File: pkg/apiserver/manager/hwameistor/localvolume_controller.go [hunk 2]
```diff
@@ -420,6 +420,7 @@ func (lvController *LocalVolumeController) CreateVolumeExpand(volName string, ta
 	if lv == nil {
 		return nil, fmt.Errorf("volume %v is not exists", volName)
 	}
+	requiredCapacityBytes := lv.Spec.RequiredCapacityBytes
```
Comment (SSmallMonster) [inline pkg/apiserver/manager/hwameistor/localvolume_controller.go:423]:
```suggestion
	currentCapacityBytes := lv.Spec.RequiredCapacityBytes
```

File: pkg/apiserver/manager/hwameistor/localvolume_controller.go [hunk 3]
```diff
@@ -438,9 +439,9 @@ func (lvController *LocalVolumeController) CreateVolumeExpand(volName string, ta
 	if err != nil {
 		return nil, err
 	}
-
-	if quantity.Value() > freeCount {
-		log.Errorf("freeCount is %d , targetCapacity is %d", freeCount, quantity.Value())
+	targetCapacityBytes := quantity.Value()
+	if targetCapacityBytes > freeCount+requiredCapacityBytes {
```
Comment (SSmallMonster) [inline pkg/apiserver/manager/hwameistor/localvolume_controller.go:443]:
```suggestion
	if targetCapacityBytes - currentCapacityBytes > freeCount {
```

**Review by SSmallMonster — APPROVED** [review]
nits

**Comment by SSmallMonster** [comment]
LGTM

**Pull request merged**